core/modules/user/user.module

```diff
@@ -1625,7 +1625,8 @@ function user_menu_alter(&$items) {
  */
 function user_admin_bar_output_alter(&$content) {
   // Admin menu shows both the parent menu item and its children.
-  // Remove the duplicate child element so "Manage fields" doesn't show up twice.
+  // Remove the duplicate child element, so "Manage fields" doesn't show up
```

Comment:
```suggestion
  // Remove the duplicate child element so "Manage fields" doesn't show up
```
The added comma is ungrammatical.

Reply:
I guess it depends from whether those clauses are considered independent or dependent. Probably they are dependent clauses as the clauses in the following sentence.

> I went to the store so I could buy tomatoes.